inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/subtract.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
After discussion with @dmitry-gorokhov, we have take a decision to make single eltwise tests file instead of existing. Dmitry had more powerful argument in our discussion.

I will use this PR in merge of these tests. Sorry for inconvenience.

Reply:
@iefode, can you please elaborate on "I will use this PR in merge of these tests" - what do you mean?

Reply:
I will refactor these chanes in the refactoring

Reply:
Refactor in the refactoring? It's still unclear. Is anything expected from our side?

Reply:
I will do it myself. I don't expected some actions from your side